packages/fluentui/react-northstar/package.json

```diff
@@ -15,6 +15,7 @@
     "@fluentui/react-icons-northstar": "^0.47.0",
     "@fluentui/react-proptypes": "^0.47.0",
     "@fluentui/styles": "^0.47.0",
+    "body-scroll-lock": "^3.0.1",
```

Comment:
@jurokapsiar @layershifter , please do we need any approvals for third-party library usage?

Reply:
Removed